.github/workflows/js.yml

```diff
@@ -73,7 +74,7 @@ jobs:
           yarn i18n:check
       - name: Run tests
         run: |
-          yarn test --silent
+          yarn test --silent ${{ matrix.option }} src/experiment-tracking/components
```

Comment:
Wondering why splitting makes this job less flaky

Reply:
yeah i'm not too sure either, i have some vague feeling that we might be running into memory issues or something, because a lot of the failures have a message like `Error: The operation was canceled.`

Reply:
I filed https://github.com/mlflow/mlflow/pull/10947 to see how much memory is being used while jest is running:

<img width="930" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/4f73a270-1a72-43b6-a896-375d6f1651e4">

Reply:
oh nice, let me add it here too so we can compare

Reply:
i think we might be leaking memory somewhere, peak memory usage for [this PR](https://github.com/mlflow/mlflow/actions/runs/7707132871/job/21003808185?pr=10944) seems to be < 5GB, and i see >6GB on your PR